src/main/java/com/openshift/internal/restclient/URLBuilder.java

```diff
@@ -42,7 +41,7 @@
 	private String baseUrl;
 	private String kind;
 	private String name;
-	private Map<String, String> params = new HashMap<String, String>();
+	private ArrayList<AbstractMap.SimpleEntry<String,String>> params = new ArrayList<>();
 	private final IApiTypeMapper typeMappings;
```

Comment:
The exec endpoint takes multiple "command" parameters. The first specifies the binary to run. Subsequent command parameters specify arguments. Changes enable this pattern in URLBuilder.